cpp/cmake/cudf-config.cmake.in

```diff
@@ -15,12 +15,22 @@ find_dependency(CUDAToolkit)
 find_dependency(Threads)
 find_dependency(ZLIB)
 
+# Don't look for a Boost CMake configuration file because it adds the
+# `-DBOOST_ALL_NO_LIB` and `-DBOOST_FILESYSTEM_DYN_LINK` compile defs
+set(Boost_NO_BOOST_CMAKE ON)
+find_dependency(Boost @CUDF_MIN_VERSION_Boost@ COMPONENTS filesystem)
+
 find_dependency(Arrow @CUDF_VERSION_Arrow@)
+
+set(ArrowCUDA_DIR "${Arrow_DIR}")
```

Comment:
Is this needed because of an Arrow bug in finding the `ArrowCUDA` project?

Reply:
Yes it is.

Reply:
We should open a JIRA issue and consider putting in a PR to try to get this fixed before 4.0.0 is released.